kolibri/plugins/learn/templates/learn/learn.html

```diff
@@ -5,7 +5,7 @@
 
 {% block content %}
   <!-- Bootstrapping the initial information for all channels. -->
-  {% kolibri_bootstrap_collection 'channel' 'ChannelResource' available="true" %}
+  {% kolibri_bootstrap_collection 'channel' 'ChannelResource' available=True %}
```

Comment:
Note that this is on the learn page, not on the content import page - is @jonboiser changing how he manages channels in the learn app or in content import?

If he is not touching the learn app, then this change must be reverted, because no frontend code will ever make a request with a kwarg value of `True` because GET params will never directly evaluable to a Boolean.

If he is changing the learn app, and all channels are being fetched, then the kwarg needs to be removed from here completely, so that the bootstrapping will actually be used.